Remotion/Web/Development.WebTesting/Utilities/RetryUntilTimeout.cs

```diff
@@ -55,27 +55,27 @@ public void Run ()
       _retryUntilTimeout.Run();
     }
 
-    public static void Run ([NotNull] Action action)
+    public static void Run ([NotNull] Action action, TimeSpan timeout = default, TimeSpan retryInterval = default)

```

Comment:
The default of a TimeSpan is `00:00:00`, which is a valid timeout/interval. I would use `TimeSpan?` here and below.